### PR TITLE
Update set-up.md

### LINKED ISSUE #12543: Broken link for setup instructions
The reference in this link no longer exists:

Use those instructions to create CNAME records described in [Simplify Windows enrollment without Azure AD Premium](https://learn.microsoft.com/en-us/mem/intune/enrollment/windows-enroll#simplify-windows-enrollment-without-azure-ad-premium).

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 67847a65-598f-f3ff-0864-92a39dfdd86d
* Version Independent ID: e47b69f4-1964-e18b-e81e-31edc2e6798f
* Content: [Set up Basic Mobility and Security - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/basic-mobility-security/set-up?view=o365-worldwide)
* Content Source: [microsoft-365/admin/basic-mobility-security/set-up.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/basic-mobility-security/set-up.md)
* Service: **microsoft-365-business**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Added the CNAME information as the old information was not available

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12543

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/basic-mobility-security/set-up.md [hunk 1]
```diff
@@ -50,7 +50,12 @@ When the service is ready, complete the following steps to finish setup.
...
 
-Need help setting up the records? Find your domain registrar and select the registrar name to go to step-by-step help for creating DNS record in the list provided in [Add DNS records to connect your domain](/office365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider). Use those instructions to create CNAME records described in [Simplify Windows enrollment without Azure AD Premium](/mem/intune/enrollment/windows-enroll#simplify-windows-enrollment-without-azure-ad-premium).
+Need help setting up the records? Find your domain registrar and select the registrar name to go to step-by-step help for creating DNS record in the list provided in [Add DNS records to connect your domain](/office365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider). Use the below details to create CNAME records
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/basic-mobility-security/set-up.md]:
```suggestion
Need help setting up the records? Find your domain registrar and select the registrar name to go to step-by-step help for creating DNS records in the list provided in [Add DNS records to connect your domain](/office365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider). Use the following details to create CNAME records:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**